gradio/blocks.py

```diff
@@ -551,8 +551,13 @@ def get_api_info(config: dict, serialize: bool = True):
                 continue
             label = component["props"].get("label", f"value_{o}")
             serializer = serializing.COMPONENT_MAPPING[type]()
-            assert isinstance(serializer, serializing.Serializable)
-            info = serializer.api_info()
+            if component.get("api_info") and after_new_format:
```

Comment:
This is the same logic applied to inputs also applied to outputs. I think it was just an oversight it wasn't done before.